org.scala-ide.sdt.core/src/org/scalaide/core/internal/extensions/AutoEdits.scala

```diff
@@ -30,7 +30,8 @@ object AutoEdits {
     fullyQualifiedName[SurroundSelectionWithString] → SurroundSelectionWithStringSetting,
     fullyQualifiedName[SurroundSelectionWithParentheses] → SurroundSelectionWithParenthesesSetting,
     fullyQualifiedName[SurroundSelectionWithBraces] → SurroundSelectionWithBracesSetting,
-    fullyQualifiedName[SurroundSelectionWithBrackets] → SurroundSelectionWithBracketsSetting
+    fullyQualifiedName[SurroundSelectionWithBrackets] → SurroundSelectionWithBracketsSetting,
+    fullyQualifiedName[SurroundSelectionWithAngleBrackets] → SurroundSelectionWithAngleBracketsSetting
```

Comment:
I would rather remove all angle-bracket auto-edits than add more (I understand it's because the logic requires it in the base class for all brackets.

Or, if you really want them for XML, can you add them only for XML partitions? I think auto-edits are usually enabled per partition type.